src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -379,6 +379,7 @@ void ImageReplayer<I>::bootstrap() {
 
   {
     Mutex::Locker locker(m_lock);
+    request->get();
```

Comment:
Okay, I understand. Just one more question. I saw that in the code before today's commits, you would explicitly delete the request object, instead of relying in RefCountedObj.  Why didn't you kept that approach?